css/components/navigation.scss

```diff
@@ -1,33 +1,100 @@
 // Navbar styles
+
 .navbar {
-  width: 85%;
+  width: 100%;
   height: auto;
-  line-height: 1.2;
   margin: 0 auto;
   display: flex;
+  flex-direction: row;
   justify-content: space-around;
+  max-height: $s__navbar-height;
+  transition: max-height 250ms;
   background-color: $c__white;
 
-  @media (max-width: $s__breakpoint--large) {
-    width: 100%;
-  }
-
-  @media (max-width: $s__breakpoint--small) {
-    flex-direction: column;
+  .navbar__hamburger {
+    display: none;
   }
 
   .navbar__link--wrapper {
     padding-top: 20px;
     padding-bottom: 20px;
+    margin-bottom: 5px;
     align-self: center;
+    background-color: $c__white;
 
     .navbar__link {
+      padding-top: 20px;
+      padding-bottom: 20px;
+      padding-left: 30px;
+      padding-right: 30px;
       text-decoration: none;
-      color: $c__grey--light;
+      color: $c__grey;
+      transition: color 200ms ease-in-out;
 
       &:hover {
         color: $c__black;
       }
     }
   }
+  
+  // handle navbar change
+  @media (max-width: $s__breakpoint--small) {
+    max-height: $s__navbar-height + 5;
+
+    .navbar__link--wrapper {
+      height: $s__navbar-height;
+      padding: 0;
+
+      .navbar__link {
+        display: none;
+      }
+    }
+    
+    &.navbar--open {
+      flex-direction: column;
+      max-height: $s__navbar-height * 7;
+
+      .navbar__link--wrapper {
+        width: 85%;
+        transition: box-shadow 200ms ease-in-out;
```

Comment:
The box-shadow looks a little strange since it's not full width, can you either make it full width or get rid of it?